www/apps/book/next.config.mjs

```diff
@@ -157,6 +157,21 @@ const nextConfig = {
         destination: "/:path*",
         permanent: true,
       },
+      {
```

Comment:
q: Any reason we shouldn't redirect to the v2 docs here?

Reply:
Good point! Fixed